<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>815</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>几何物体3D设计</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Images/prefs.png</normaloff>Images/prefs.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{	
	font-family:微软雅黑;
	font-size:14px;
	color: #1d649c;
}

QGroupBox{
   font-family:微软雅黑;
   font-size:15px;
   color: rgb(126, 0, 0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="childrenCollapsible">
       <bool>true</bool>
      </property>
      <widget class="QTreeWidget" name="notesTree">
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="headerHidden">
        <bool>false</bool>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="headerVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="headerCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="headerHighlightSections">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">项目操作菜单</string>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
          <underline>false</underline>
          <kerning>false</kerning>
         </font>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="background">
         <color>
          <red>235</red>
          <green>235</green>
          <blue>235</blue>
         </color>
        </property>
        <property name="foreground">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </property>
       </column>
      </widget>
      <widget class="QMdiArea" name="mdiArea"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>923</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actionAddSubFolder"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>配置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>项目类型</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>后台对接</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>数据导入导出</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionAddSibleFolder"/>
   <addaction name="actionDelete"/>
   <addaction name="actionAddNote"/>
   <addaction name="action"/>
   <addaction name="separator"/>
   <addaction name="actionAddSubFolder"/>
   <addaction name="action_5"/>
   <addaction name="action_3"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="action_2"/>
  </widget>
  <action name="actionAddSubFolder">
   <property name="icon">
    <iconset>
     <normaloff>Images/subfolder.png</normaloff>Images/subfolder.png</iconset>
   </property>
   <property name="text">
    <string>添加子目录</string>
   </property>
   <property name="toolTip">
    <string>添加子目录</string>
   </property>
  </action>
  <action name="actionAddNote">
   <property name="icon">
    <iconset>
     <normaloff>Images/note.gif</normaloff>Images/note.gif</iconset>
   </property>
   <property name="text">
    <string>添加笔记</string>
   </property>
   <property name="toolTip">
    <string>添加笔记</string>
   </property>
  </action>
  <action name="actionAddSibleFolder">
   <property name="icon">
    <iconset>
     <normaloff>Images/siblefolder.png</normaloff>Images/siblefolder.png</iconset>
   </property>
   <property name="text">
    <string>添加兄弟目录</string>
   </property>
   <property name="toolTip">
    <string>添加兄弟目录</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>Images/save.png</normaloff>Images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存目录树</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset>
     <normaloff>Images/folder.png</normaloff>Images/folder.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset>
     <normaloff>Images/replace.png</normaloff>Images/replace.png</iconset>
   </property>
   <property name="text">
    <string>操作3</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset>
     <normaloff>Images/prefs.png</normaloff>Images/prefs.png</iconset>
   </property>
   <property name="text">
    <string>操作5</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset>
     <normaloff>Images/delayCalc.png</normaloff>Images/delayCalc.png</iconset>
   </property>
   <property name="text">
    <string>的说法</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset>
     <normaloff>Images/Question22.png</normaloff>Images/Question22.png</iconset>
   </property>
   <property name="text">
    <string>是否</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
